Lib/test/test_ctypes/test_refcounts.py

```diff
@@ -97,5 +97,31 @@ def func(a, b):
         f(1, 2)
         self.assertEqual(sys.getrefcount(ctypes.c_int), a)
 
+    @support.refcount_test
+    def test_callback_py_object_none_return(self):
+        """Test that returning ``None`` from a ``py_object`` callback
+        does not affect ``None``'s refcount (bpo-36880)."""
+
+        import sys
```

Comment:
```suggestion
        import sys

        # Increase the initial refcount to avoid crashing due to a regression.  
        refcount_margin = [None] * 20000  
```

Reply:
This doesn't prevent the crash, it just delays it until after the test case finishes (when `refcount_margin` is deallocated). I think the behavior is better without this addition, because then the faulthandler stack trace points at the correct test case that caused the crash.